fleetspeak/src/client/internal/monitoring/resource_usage_fetcher_unix.go

```diff
@@ -42,6 +44,12 @@ type ResourceUsage struct {
 
 	// Resident set size for a process.
 	ResidentMemory int64
+
+	// Client IO data read so far, in bytes.
+	IORead int64
```

Comment:
Same comment about the names here. I'd suggest `IOReadBytes` and `IOWriteBytes`.